sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -86,19 +94,21 @@ def set_authentication_policy(credential, kwargs):
     elif hasattr(credential, "get_token"):
         if not kwargs.get("authentication_policy"):
             if kwargs.get("region") and kwargs.get("resource_id"):
+                scope = kwargs.pop("audience", DEFAULT_ENTRA_ID_SCOPE) + DEFAULT_SCOPE
                 kwargs["authentication_policy"] = TranslatorEntraIdAuthenticationPolicy(
                     credential,
                     kwargs["resource_id"],
                     kwargs["region"],
-                    kwargs.pop("scopes", DEFAULT_ENTRA_ID_SCOPE),
+                    scope,
                 )
             else:
                 if kwargs.get("resource_id") or kwargs.get("region"):
                     raise ValueError(
-                        "Both 'resource_id' and 'region' must be provided with a TokenCredential for authentication."
+                        """Both 'resource_id' and 'region' must be provided with a TokenCredential for
+                         regional resource authentication."""
                     )
                 kwargs["authentication_policy"] = BearerTokenCredentialPolicy(
-                    credential, *kwargs.pop("scopes", [DEFAULT_TOKEN_SCOPE]), kwargs
+                    credential, *kwargs.pop("audience", [DEFAULT_TOKEN_SCOPE]), kwargs
```

Comment:
Audience is a string - so cannot be used with * syntax here.
I think you would need something like:
`*[kwargs.pop("audience", DEFAULT_TOKEN_SCOPE)]`

Reply:
Also - shouldn't that be `**kwargs` at the end?